app/assets/javascripts/dac.js

```diff
@@ -22,8 +22,7 @@ if(backLink){
   });
 }
 
-var printLink = document.getElementById('printLink');
-if(printLink){
+if (document.getElementById('printLink')){
     printLink.addEventListener('click', function(e){
```

Comment:
that name wont work anymore. Needs to be document.getElementById('printLink').addEvent...